### DISCUSSION TITLE
Please add support for SQLAlchemy 1.4.54 on aarch64/Python 3.13

### DISCUSSION THREAD

**Discussion by heikkiorsila** [sqlalchemy/sqlalchemy#12091] [Ideas]
SQLAlchemy 1.4.54 seems not to be supported for Python 3.13 on manylinux2014_aarch64. The following install command fails when I'm trying to package a distribution with SQLAlchemy:
``` pip install sqlalchemy==1.4.54 --only-binary=:all: --platform manylinux2014_aarch64 --python-version 3.13 --target dist
...
ERROR: No matching distribution found for sqlalchemy==1.4.54
```
The above works for Python version 3.12. Could you please provide an upgrade, thanks!

**Comment by CaselIT** [comment]
Hi,

1.4 was not updated to support 3.13. There is a source distribution so assuming everything works you should be able to install that.
I would suggest to either upgrade to 2.0 or use 3.12

**Reply by jgentil** [reply]
Is this true? Because the release notes for 1.4.53 state that 3.13 compatibility was added. 
https://docs.sqlalchemy.org/en/20/changelog/changelog_14.html#change-1.4.53

Can you clarify?

**Reply by CaselIT** [reply]
I guess I forgot about that. I think there is no specific wheel for 3.13 though

**Reply by jgentil** [reply]
OK, thanks for the reply!  We will build 3.13 wheels locally then.
